src/locales/en-CA.ts

```diff
@@ -2,7 +2,6 @@ import { TRomcalLocale } from "../models/romcal-locale";
 
 export default {
     sanctoral: {
-        saintsJeanDeBrebeufAndIsaacJoguesPriestsAndCompanionsMartyrsSaintPaulOfTheCrossPriest:
-            "Saints John de Brébeuf, Isaac Jogues, Priests, and Companions, Martyrs, Secondary Patrons of Canada",
+        saintsJeanDeBrebeufAndIsaacJoguesPriestsAndCompanionsMartyrsSaintPaulOfTheCrossPriest: "Saints John de Brébeuf, Isaac Jogues, Priests, and Companions, Martyrs, Secondary Patrons of Canada",
```

Comment:
We really don't have to manually edit instances such as these because `prettier` will do the work for us on commit (or directly in our IDE's if we have the correct setup). If the outcome of the auto formatter is not desirable, we can agree on new settings that do. In this particular instance, the setting that is in question is [printWidth](https://prettier.io/docs/en/options.html#print-width). The documentation recommends a print width of 80. I opted for 120 because of the nature of our code. However, we can agree on a wider width if this is insufficient (as in the case of this celebration). However, I would advise against this as _overall_ legibility of the code will be impacted (especially in smaller screens) and that cases such as these are rare in the context of our code base.

Reply:
Okay, I get it.

When I separate the celebrations occurring on the same day, the keys and their values should be shorter. Then we might want to change the `printWidth` to the recommended `80`.

> However, I would advise against this as overall legibility of the code will be impacted (especially in smaller screens) and that cases such as these are rare in the context of our code base.

This depends on the wrapping. I always use soft-wrapping and then this is not an issue—not even on the phones. :smiley:

Reply:
I'm fine with the current `printWidth: 120`, I think this value is a good balance for our codebase. Because reducing the width too much often doesn't help to make code readable.